GEOSwiftMapKitTests/GEOSwift+MapKitTests.swift

```diff
@@ -92,6 +96,34 @@ final class MapKitTests: XCTestCase {
         XCTAssertEqual(mkPolygon.interiorPolygons?.first?.pointCount, 4)
     }
 
+    func testCreateMKMultiPolylineFromMultiLineString() {
+        guard #available(iOS 13.0, tvOS 13.0, macOS 10.15, *) else {
```

Comment:
Usually I'd just add the @available attribute to the containing method itself, but for some reason that was insufficient in this case. The test still ran on CI for macOS even though the host platform was macOS 10.14. I haven't had time to investigate why, but this approach does work.